tests/WdlV1_1.py

```diff
@@ -0,0 +1,146 @@
+f = open("Qc.wdl", "r")
+
+import sys
+from antlr4 import *
+from WdlV1_1Lexer import WdlV1_1Lexer
+from WdlV1_1Parser import WdlV1_1Parser
+from WdlV1_1ParserVisitor import WdlV1_1ParserVisitor
+import cwl_utils.parser_v1_2 as cwl
+
+from ruamel import yaml
+
+text = InputStream(f.read())
+lexer = WdlV1_1Lexer(text)
+stream = CommonTokenStream(lexer)
+parser = WdlV1_1Parser(stream)
+tree = parser.document()
+
+#create WdlV1_1ParserVisitor object and return all inputs, outputs, etc
+ast = WdlV1_1ParserVisitor()
+ast.walk_tree(tree)
+
+#returns the entire command including "command{........}"
+command = ast.task_command
+command = command[command.find("{")+1:-1] #removing the command{} part
+command = command.strip().split("\\") #split by '\'
+
+base_command = ""
+command_arguments = []
+
+for a in command:
+    #if it contains = , it's taken as an argument
+    if "=" in a or "~" in a:
+        command_arguments.append(a.strip())
+    #else add to base command
+    else:
+        base_command+=a
+
+#split the base command by spaces
+base_command = base_command.split()
+
+## get command arguments
+index = 0
+for i in command_arguments:
+    if "~" in i:
+        parameter_reference = i[i.find("~{")+2:i.find("}")] #finding the name of the parameter
+        sub_str = i.strip().split("~")
+        if "INPUT" not in sub_str[0]:
+            command_arguments[index] = sub_str[0]+"$(inputs."+parameter_reference+")"
+        else:
+            command_arguments[index] = sub_str[0]+"$(inputs."+parameter_reference+".path"")"
+    index+=1
```

Comment:
Move this into the `main` function. Add an argument for the file name of the input WDL file